src/services/balancer.js

```diff
@@ -23,7 +22,7 @@ export default class Balancer {
     this.subgraphUrl = process.env.REACT_APP_SUBGRAPH_URL
     this.gasBase = GAS_BASE
     this.gasPerSwap = GAS_PER_SWAP
-    this.maxSwaps = MAX_SWAPS
+    this.maxSwaps = process.env.BALANCER_MAX_SWAPS
```

Comment:
I think better move all configuration to one config file and then take data from this file, because so often need prepare data before using. Like example, sometimes need check value on undefined or convert to type what will used later in code. So, using env variables like this look bad.